bpf/bpf_lxc.c

```diff
@@ -500,7 +512,7 @@ static inline int handle_ipv4_from_lxc(struct __sk_buff *skb, __u32 *dstID)
 		 */
 		ct_state_new.src_sec_id = SECLABEL;
 		ret = ct_create4(get_ct_map4(&tuple), &tuple, skb, CT_EGRESS,
-				 &ct_state_new);
+				 &ct_state_new, verdict > 0);
```

Comment:
[0] This is in handling packets "from_lxc", ie from the endpoint, so the verdict here (if nonzero) is an egress L7 policy.

Reply:
See above